index.js

```diff
@@ -71,6 +71,7 @@ module.exports = {
         "no-labels": 2,
         "no-lone-blocks": 2,
         "no-loop-func": 2,
+        "no-multiple-empty-lines": [2, { max: 1 }],
```

Comment:
Yup I'm open to enforcing this.
maxEOF and maxBOF set to 0? Any thoughts @andrinealver @bobbyrenwick @braposo ?